docs/src/lib/indexing.md

```diff
@@ -79,3 +79,19 @@ For performance reasons, accessing, via `getindex` or `view`, a single `row` and
 ## `setindex!`
 
 Under construction
+
+## Broadcasting
+
+`AbstractDataFrame` is converted to a matrix in broadcasting.
+`DataFrameRow`, `GroupedDataFrame`, `DataFrameRows` and `DataFrameColumns` are `collect`ed in broadcasting.
+
+Additionally it is possible to assign a value to `AbstractDataFrame` and `DataFrameRow` using the `.=` operator.
+In such an operation `AbstractDataFrame` is two dimensional and `DataFrameRow` as a single dimensional.
+
+If column indexing using symbols is performed the order of columns in the operation is specified by the order of symbols.
+
+It is allowed to perform `df[col] .= value` even if `col` is not present in the `DataFrame` and it is a `Symbol`.
+In such a case a new column will be created.
+A data frame with zero columns is always considered to have zero rows.
+The assignment broadcasting is not supported for `df.col` style.
```

Comment:
It isn't?  Or are you just stating that the special column-creation behavior doesn't work?

Reply:
`df.col` does not work in the column-creation case. I will state it more clearly (and maybe even we will then drop it if we decide not to support column creation).